src/shared/components/msktable/MSKTable.tsx

```diff
@@ -18,12 +21,20 @@ type MSKTableProps<T> = {
 
 class MSKTableStore<T> {
     @observable public filterString:string;
-    @observable public page:number;
+    @observable private _page:number;
     @observable public itemsPerPage:number;
     @observable public sortColumn:string;
     @observable public sortAscending:boolean;
     @observable public columns:Column<T>[];
```

Comment:
I should've reviewed this one on the previous PR, but can we make `columns` an object instead of an array (like we did in `EnhancedReactTable`)? It is easier to customize an object than an array.